internal/core/openapi/legacy/auth/user.go

```diff
@@ -140,27 +140,11 @@ func (u *User) get(req *http.Request, state GetUserState) (interface{}, AuthResu
 		if state == GotInfo {
 			return u.info, AuthResult{AuthSucc, ""}
 		}
-		// 1. 如果 request.Header 中存在 'ORG', 直接使用它作为 OrgID
-		// 2. 否则 使用 request.Host 来查询 OrgID
 		orgHeader := req.Header.Get("ORG")
-		var orgID uint64
-		var noOrgID bool
-		if orgHeader != "" && orgHeader != "-" {
-			org, err := u.bundle.GetOrg(orgHeader)
-			if err != nil {
-				return nil, AuthResult{InternalAuthErr, err.Error()}
-			}
-			orgID = org.ID
-		} else {
-			domain := strutil.Split(req.Host, ":")[0]
-			org, err := u.bundle.GetDopOrgByDomain(domain, string(u.info.ID))
-			if err != nil {
-				return nil, AuthResult{InternalAuthErr, err.Error()}
-			} else if org == nil {
-				noOrgID = true
-			} else {
-				orgID = org.ID
-			}
+		domainHeader := req.Header.Get("domain")
```

Comment:
define const for header `org` and `domain`.
and it's ugly the one is uppercase while another is lowercase.